net/blockwise/blockwise.go

```diff
@@ -462,8 +430,8 @@ func (b *BlockWise[C]) Handle(w *responsewriter.ResponseWriter[C], r *pool.Messa
 		b.errors(fmt.Errorf("continueSendingMessage(%v): %w", r, err))
 		return
 	}
-	if b.autoCleanUpResponseCache && !more {
-		b.RemoveFromResponseCache(token)
+	if !more && sendingMessageCached.Data().Code() > codes.DELETE {
```

Comment:
Are codes `>` than `codes.DELETE` invalid or why are only those codes deleted from cache?

Reply:
For codes GET,POST,PUT,DELETE, we want them to wait for pairing response and then delete them when the full response comes in or when timeout occurs.